integration-tests/features/stack_analyses.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Jobs API
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I access /api/stack-analyses
+    Then I should get 404 status code
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I wait 10 seconds
+    Then I wait 10 seconds
```

Comment:
Ok, that's what I was thinking.
But as I understand it, both `When I wait 10 seconds` and `Then I wait 10 seconds` do `time.sleep(10)`, so why we have both these lines here ? For example only `Then I wait 20 seconds` would do the same, wouldn't it ?